pkg/autoscaler/multiscaler.go

```diff
@@ -38,55 +38,76 @@ const (
 )
 
 type Metric struct {
+	metav1.ObjectMeta
+	Spec   MetricSpec
+	Status MetricStatus
+}
+
+type MetricSpec struct {
+	TargetConcurrency float64
+}
+
+type MetricStatus struct {
 	DesiredScale int32
 }
 
-// UniScaler records statistics for a particular KPA and proposes the scale for the KPA's target based on those statistics.
+func (m *Metric) clone() *Metric {
```

Comment:
There are a number of places that we run `deepcopy-gen` even for non-CRD structs

Reply:
Oh yeah!  Done.